pypsa/plot.py

```diff
@@ -605,6 +605,12 @@ def _get_coordinates(network, layouter=None):
         return network.buses["x"], network.buses["y"]
 
 
+_token_required_mb_styles = ['basic', 'streets', 'outdoors', 'light', 'dark',
+                    'satellite', 'satellite-streets']
+
+_open__mb_styles = ['open-street-map', 'white-bg', 'carto-positron',
+                 'carto-darkmatter', 'stamen-terrain', 'stamen-toner', 'stamen-watercolor']
```

Comment:
formatting could be tidier.

Reply:
I'm trying to figure out what you mean...

Reply:
Nothing major. My suggestion is to align the multiline lists:
```python
_token_required_mb_styles = ['basic', 'streets', 'outdoors', 'light', 'dark',
                             'satellite', 'satellite-streets']

_open__mb_styles = ['open-street-map', 'white-bg', 'carto-positron',
                    'carto-darkmatter', 'stamen-terrain', 'stamen-toner', 'stamen-watercolor']
```

Reply:
makes sense